protocols/relay/examples/docker-compose.yml

```
@@ -0,0 +1,53 @@
+version: '3.4'
+services:
+  relay: 
+    image: libp2p-relay
+    environment:
+      - relay_command_arg_env=relay
+      - deterministic_peer_env=1
+      - address_env=/ip6/::/tcp/4444
```

```suggestion
    command:
      - "./relay"
      - "--mode=relay"
      - "-d=1"
      - "--address=/ip6/::/tcp/4444"
```


Why this additional indirection through environment variables? Why not pass these as commandline arguments to the binary directly?

---

Agree , please see the next change